src/ripe/order.py

```diff
@@ -82,6 +82,26 @@ def log_order(self, number):
         contents = self.get(url)
         return contents
 
+    def chat_order(self, number):
+        url = self.base_url + "orders/%d/chat" % number
+        contents = self.get(url)
+        return contents
+
+    def chat_lines_order(self, number, *args, **kwargs):
+        url = self.base_url + "orders/%d/chat/lines" % number
+        contents = self.get(url, **kwargs)
+        return contents
+
+    def chat_lines_count_order(self, number):
+        url = self.base_url + "orders/%d/chat/lines/count" % number
+        contents = self.get(url)
+        return contents
+
+    def chat_create_line_order(self, number, content):

```

Comment:
```suggestion
    def create_chat_line_order(self, number, content):

```

Reply:
Formatting like: https://github.com/ripe-tech/ripe-api/blob/jc/4702-general-order-chat/src/ripe/order.py#L130

Reply:
Yes, it should. I ported those from the SDK but the effort to change these everywhere might not be worth